src/background/initialize.ts

```diff
@@ -38,6 +40,8 @@ export async function initialize() {
     setUninstallURL();
   });
   initializeAnalytics({ account });
+  initializeRegisterChain(account);
+  initializeRegisterAddresses();
```

Comment:
Do these handlers need separate initializations?

Btw their functionality looks a lot like analytics. Can they be added to the current `analytics.background.ts` file?

Reply:
I don't want to mix analytics and lite-chain support. However, it is ok to combine them together